src/diffpy/utils/tools.py

```diff
@@ -91,47 +90,62 @@ def _create_global_config(args):
     return return_bool
 
 
-def get_user_info(args=None):
+def get_user_info(owner_name=None, owner_email=None, owner_orcid=None):
```

Comment:
much better - instead of `args`, this line explicitly communicates what info can be optionally provided